lib/fog/utilities.rb

```diff
@@ -1,7 +1,9 @@
 class Hash
   def symbolize_keys!
     keys.each do |key|
-      self[(key.to_sym rescue key)] = delete(key) if key.respond_to?(:to_sym) && !key.is_a?(Fixnum)
+      if key.respond_to?(:to_sym) && !key.is_a?(0.class)
+        self[(key.to_sym rescue key)] = delete(key)
```

Comment:
Ah good spotting. In my zeal to fix the Fixnum issue I ignored the glaring rescue error.